setup.py

```diff
@@ -113,7 +114,7 @@ def alltests():
           'Topic :: Internet :: WWW/HTTP',
           'Framework :: Zope3',
           ],
-      url='http://pypi.python.org/pypi/zope.container',
+      url='http:///zopecontainer.readthedocs.io',
```

Comment:
The project homepage link takes on increasing prominence in Warehouse (the PyPI successor), being placed right at the top. It makes sense to me that a visitor from warehouse would click the link to get to documentation about the project, not a github page that doesn't really tell them how to use the package. 

I do agree that whatever that link goes, you should be able to arrive at the alternate location easily. In other projects, I've handled this by having the RTD homepage have a direct link to the github project in a 'development' section (where I also put the CI badges). Having a RTD badge in README.rst can also accomplish this, but it's one more step for a user viewing the PyPI page and looking for documentation.